### DISCUSSION TITLE
Octopus Saving Sessions on Hubitat

### DISCUSSION THREAD

**Discussion by Automatic-Bob** [BottlecapDave/HomeAssistant-OctopusEnergy#736] [General]
Hi!
I am a complete novice on coding. I have a Hubitat that controls my Tesla Powerwalls and I am looking for an app that will run on the Hubitat to automatically join the Octopus Saving Sessions. Are you able to help or point me in the right direction?

**Comment by BottlecapDave** [comment]
Sorry for the late response. Unfortunately my knowledge of hubitat and its codebase is non existent. You will probably be best asking in the hubitat forums.

The graphql mutation can be found at https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/custom_components/octopus_energy/api_client/__init__.py#L241

The graphql query for retrieving the list of events can be found at https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/custom_components/octopus_energy/api_client/__init__.py#L253

**Comment by Automatic-Bob** [comment]
Hi! Dave,

I have recently received a Home Mini and have looked to see how I can edit your Octopus Energy integration, but, I have not found a way. Please advise if I need to remove it and reinstall or if there is a way to introduce the Home Mini?

Regards
Bob.

**Comment by Dukeicon** [comment]
If you already have the Octopus intergration set up, you just need to tick the checkbox to say that you have a Home Mini.

Settings > Devices & Services > Intergrations > Octopus Energy > Configure
Tick the 'I have a Home Mini' checkbox and then click 'Submit' at the bottom.

<img width="504" alt="Home Mini - Configuration Dialogue" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/115842708/b22a27e3-5f39-40e2-9d60-2704194f128d">

You'll probably need to restart Home Assistant (although i'm not 100% sure on that).

More info on the Home Mini can be found here: https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/account/#home-mini

**Comment by Automatic-Bob** [comment]
Thank you,
